### ISSUE TITLE
Apple pass generation certificate expired

### ISSUE THREAD

**Issue by sargant** [sargant/dfyb.run#52]
Need to stump up £79 for a renewal 😬

**Comment by moxus** [comment]
Since Android seems to care less about apples certificates, the pkpass-files would work fine there (tested with walletpasses.io). Pull-Request #53 disabled it for everyone.

**Comment by sargant** [comment]
Interesting! I haven't really tested on Android at all, as I was focusing on the end-to-end flow for iOS, particularly for the Apple Watch. Is it an easy workflow on Android, and for things like the Pixel Watch?

I'm not entirely sure how I generate passes without a signing certificate though - is just the zipped up bundle without a certificate sufficient?

**Comment by moxus** [comment]
Not sure about the watch, but it works on the phone. I opened PR #54
It does not need any changes besides not disabling the buttons. It will be sigend with the current certificates but they are ignored, so thats not an issue. The node library from Apple does not support creating pkpass files without certs (so i could not completely test it), but it seems to work with your outdated certificate. I tested it with a link against the current live /api/generate
